### PR TITLE
Missing events domain and endpoint

### PR DESCRIPTION
**Pull request by jsilva-daitan**
Included Domain and Endpoint

### CODE REVIEW

File: bandwidth_sdk/models.py [hunk 1]
```diff
@@ -1577,6 +1577,265 @@ def send_batch(cls):
...
+        resp = client.post(cls._path, data=p_data)
+        domain_id = get_location_id(resp)
+        data.update({'id': domain_id})
+        return cls(data=data)
+
+    @classmethod
+    def list(cls, page=0, size=25):
+        """
+        :page: Used for pagination to indicate the page requested for querying a list of domains.
+        If no value is specified the default is 1.
```
Comment (lfavaro-daitan) [inline bandwidth_sdk/models.py]: default is 1 or 0?

File: bandwidth_sdk/models.py [hunk 2]
```diff
@@ -1577,6 +1594,265 @@ def send_batch(cls):
...
+        """
+        Delete endpoint instance on catapult side.
+        :return: True if it's deleted
+        """
+        client = self.client or get_client()
+        url = 'domains/{}/endpoints/{}'.format(self.domain_id, self.id)
+        client.delete(url)
+        return True
+
+    def create_token(self, **params):
```
Comment (lfavaro-daitan) [inline bandwidth_sdk/models.py]: Would move this method to endpointtoken resource, or at least create a method there that allows token creation there as well

Comment (jsilva-daitan) [inline bandwidth_sdk/models.py]: Adding on both

**Comment by lfavaro-daitan** [comment]
Approved

**Pull request merged**